pkgs/development/python-modules/labelbox/default.nix

```diff
@@ -0,0 +1,42 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, requests
+, jinja2
+, pillow
+, rasterio
+, shapely
+, pytest
+}:
+
+buildPythonPackage rec {
+  pname = "labelbox";
+  version = "2.0.1";
+
+  src = fetchFromGitHub {
+    owner = "Labelbox";
+    repo = "labelbox-python";
+    rev = "f55ba704d1becd53e62f4a46de31684ab1745c23";
+    sha256 = "sha256:09jprbqlm5sz5gg150rsw35kqnvcanp9idaih3gdxjfcj9s9lbhz";
+  };
+
+  propagatedBuildInputs = [ jinja2 requests pillow rasterio shapely ];
+
+  prePatch = ''
+    substituteInPlace requirements.txt \
+      --replace "simplification==0.4.2" ""
+    substituteInPlace setup.cfg \
+      --replace "simplification" "" 
+  '';
+
+  checkPhase =''pytest'';
+  checkInputs = [ pytest ];
+  doCheck = false;   
```

Comment:
Put in a comment as to why you are disabling the tests.

Reply:
Since the tests aren't working from the Github source anyway, maybe we can revert back to PyPi? @jonringer.

Reply:
I have reverted this back to PyPi.